src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -238,11 +238,13 @@ private void CheckNotDisposed()
                 // The conditions are ordered with stream first as that would be the most common mode
                 if (_output == null)
                 {
-                    throw new ObjectDisposedException(nameof(Utf8JsonWriter));
+                    ThrowObjectDisposedException();
                 }
             }
         }
 
+        private void ThrowObjectDisposedException() => throw new ObjectDisposedException(nameof(Utf8JsonWriter));
```

Comment:
Thanks. For consistency this method should live inside the `ThrowHelpers.cs` class

https://github.com/dotnet/runtime/blob/e05f79b11cb9dbe5a75723109fe13fd9043a2afc/src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.cs#L10